src/gui/mainwindow.cpp

```diff
@@ -1784,67 +1784,50 @@ void MainWindow::on_actionRSSReader_triggered()
 void MainWindow::on_actionSearchWidget_triggered()
 {
     if (!m_hasPython && m_ui->actionSearchWidget->isChecked()) {
-        int majorVersion = Utils::ForeignApps::pythonInfo().version.majorNumber();
+        const Utils::ForeignApps::PythonInfo pyInfo = Utils::ForeignApps::pythonInfo();
 
-        // Check if python is already in PATH
-        if (majorVersion > 0) {
-            // Prevent translators from messing with PATH
-            Logger::instance()->addMessage(tr("Python found in %1: %2", "Python found in PATH: /usr/local/bin:/usr/bin:/etc/bin")
```

Comment:
Had you really deleted this ugly message? Or just moved appropriate code in other place?

Reply:
This one is deleted because we already have a similar one at https://github.com/qbittorrent/qBittorrent/pull/9575/files#diff-b87643aecaa6077fcb7dfa7edae4d7bdR80

Reply:
>This one is deleted because we already have a similar one

The new one is better. The old one pollutes my log with "kilometer-long" PATH value.